### PR TITLE
Fixes #8307 - allow filtering of Errata by available/installable.

### PR DESCRIPTION
**Pull request by waldenraines**
Also, by default only show Errata with affected Content Hosts and allow
users to see all Errata if they so desire.

http://projects.theforeman.org/issues/8307

### CODE REVIEW

File: app/controllers/katello/api/v2/errata_controller.rb [hunk 1]
```diff
@@ -29,6 +31,13 @@ def index
 
     def custom_index_relation(collection)
       collection = filter_by_cve(params[:cve], collection) if params[:cve]
+      if params[:errata_restrict_applicable] && params[:errata_restrict_applicable].to_bool == true
```
Comment (jlsherrill) [inline app/controllers/katello/api/v2/errata_controller.rb]: no need for the '== true'

File: app/controllers/katello/api/v2/errata_controller.rb [hunk 2]
```diff
@@ -29,6 +31,13 @@ def index
...
+        collection = collection.applicable_to_systems(System.readable)
+      end
+
+      if params[:errata_restrict_installable] && params[:errata_restrict_installable].to_bool == true
```
Comment (jlsherrill) [inline app/controllers/katello/api/v2/errata_controller.rb]: same here

**Comment by waldenraines** [comment]
@jlsherrill updated.

**Comment by jlsherrill** [comment]
I'm guessing this requires https://github.com/Katello/bastion/pull/36  ?  Cause i'm not seeing a difference :)

**Comment by waldenraines** [comment]
@jlsherrill yes, sorry, meant to reference that PR here.

**Comment by jlsherrill** [comment]
I'm not a huge fan of using up that much vertical space and I don't think that we need that much text for the filters.  something like:

[ ] applicable   [ ] Installable     

With tooltips explaining, would be find i would think?   I'm not sure if its possible, but could we put these beside the search box instead of above it?

**Comment by waldenraines** [comment]
@jlsherrill updated.

**Comment by jlsherrill** [comment]
APJ

**Pull request merged**